test/cobalt/experiments/test_rag_quality.py

```
@@ -0,0 +1,245 @@
+            mock_rocketride_client.run_pipeline.return_value = {
+                'status': 'completed',
+                'output': rag_response,
+            }
+
+            retrieved_text = ' '.join(rag_response['retrieved_docs'])
+            expected_text = ' '.join(item['documents'])
+
+            evaluation = evaluate_relevance(retrieved_text, expected_text)
+            assert evaluation['score'] >= 0.4, f'Retrieval for "{item["query"]}" has low relevance: {evaluation["reasoning"]}'
+
+    def test_retrieval_coverage(self, mock_rocketride_client, sample_rag_dataset):
```

Ruff (0.15.7)

[warning] Unused method argument: `mock_rocketride_client`

(ARG002)